app/assets/javascripts/admin/modules/document-history.js

```diff
@@ -0,0 +1,38 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function DocumentHistory (module) {
+    this.module = module
+  }
+
+  DocumentHistory.prototype.init = function () {
+    this.setupEventListeners()
+  }
+
+  DocumentHistory.prototype.setupEventListeners = function () {
+    var documentHistoryTab = this.module.closest('.govuk-tabs').querySelector('#history_tab')
+
+    documentHistoryTab.addEventListener('click', function (e) {
+      var newerLink = documentHistoryTab.querySelector('.app-view-document-history-tab__newer-pagination-link')
+      var olderLink = documentHistoryTab.querySelector('.app-view-document-history-tab__older-pagination-link')
+
+      if (newerLink && newerLink.contains(event.target)) {
+        e.preventDefault()
+
+        window.fetch(new URL(document.location.origin + newerLink.dataset.remotePagination))
+        .then(function (response) { return response.text()})
+        .then((html) => { documentHistoryTab.innerHTML = html })
```

Comment:
Looks like you've got ES6 JS in this, which won't work in IE and I expect will fail uglify compliation.